libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationPerformanceEvent.java

```diff
@@ -25,16 +26,23 @@
   private final String sessionId;
   private final List<Counter> counters;
   private final List<Attribute> attributes;
+  NavigationPerformanceMetadata metadata;
 
-  NavigationPerformanceEvent(String sessionId, String eventName) {
+  NavigationPerformanceEvent(String sessionId, String eventName,
+                             NavigationPerformanceMetadata metadata) {
     this.event = PERFORMANCE_TRACE;
     this.created = obtainCurrentDate();
     this.sessionId = sessionId;
     this.counters = new ArrayList<>();
     this.attributes = new ArrayList<>();
+    this.metadata = metadata;
     attributes.add(new Attribute(EVENT_NAME, eventName));
   }
 
+  private String getScreenDensity(Context context) {
```

Comment:
Noticed that we're not adding the screen density as attribute anymore, was that on purpose? If so, we should remove this method, right?

Reply:
Yes it was on purpose, I don't think it's a good idea to add added complexity for an attribute we're not sure we'll even need.